src/@types/uia.ts

```diff
@@ -14,16 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { type AuthDict, type IAuthData } from "../interactive-auth.ts";
+import { type AuthDict } from "../interactive-auth.ts";
 
 /**
  * Helper type to represent HTTP request body for a UIA enabled endpoint
  */
 export type UIARequest<T> = T & {
     auth?: AuthDict;
 };
-
-/**
- * Helper type to represent HTTP response body for a UIA enabled endpoint
- */
-export type UIAResponse<T> = T | IAuthData;
```

Comment:
wondering if we should (temporarily) make this 

```
/** @deprecated */
export type UIAResponse<T> = T;
```

... to avoid having to synchronise the EW and js-sdk PRs.

No strong feelings though.

Reply:
We shouldn't need to synchronise them, EW can land first, shouldn't need to both land at once.

Reply:
ah nice.

Reply:
Actually no, you're right, that is a better approach and will avoid a breaking change